src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -295,6 +305,18 @@ public function configureOptions(OptionsResolver $resolver)
             return $options['expanded'];
         };
 
+        $choiceTranslationDomain = function (Options $options) {
+            return $options['translation_domain'];
+        };
+
+        $choiceTranslationDomainNormalizer = function (Options $options, $choiceTranslationDomain) {
+            if (true === $choiceTranslationDomain) {
+                return;
```

Comment:
I think using ``true`` for the case where it reuse the ``translation_domain`` variable might be better actually